src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -180,6 +220,7 @@ ensogl::define_endpoints! {
         set_disabled        (bool),
         set_input_connected (span_tree::Crumbs,Option<Type>,bool),
         set_expression      (Expression),
+        set_error_status    (Option<Error>),
```

Comment:
why not just `set_error`, like `set_expression` above, and the `set_error` function in model?

Reply:
To indicate that this can also clear the error status by setting `None`. Just `set_error` does not imply that.

Reply:
call note: `set_error`